src/index.ts

```diff
@@ -96,6 +103,13 @@ const extension: JupyterFrontEndPlugin<void> = {
     let reportCellExecutionTime = true;
     let reportCellNumber = true;
     let cellNumberType = 'cell_index';
+    const cellExecutionMetadataTable: {
+      [cellId: string]: ICellExecutionMetadata;
+    } = {};
+    const recentNotebookExecutionTimes: LRU<string, Date> = new LRU({
+      max: 500,
+      maxAge: 1000 * 60 * 60 * 10 // 10 hours
```

Comment:
There are some use-cases, like training a deep-learning model, that can take multiple days for execution.
It is difficult to estimate an appropriate maxAge in advance, so if there are no negative consequences to not setting the `maxAge`, why not leave `maxAge` unset?

Reply:
yeah I think not setting `maxAge` is fine since least recently used items will be removed anyway.